client/src/views/Login.tsx

```diff
@@ -0,0 +1,47 @@
+import { useNavigate } from 'react-router-dom'
+
+export const Login = () => {
+    const navigate = useNavigate()
+    const isSignedIn = () => (localStorage.getItem('email') ? true : false)
+
+    if (isSignedIn()) {
+        localStorage.clear()
```

Comment:
I think we want to clear out local storage only on logout
also this would clear ALL localStorage, you want to be specific with the keys usually